### ISSUE TITLE
TypeScript types do not match README usage

### ISSUE THREAD

**Issue by kylegoetz** [eleith/emailjs#302]
```typescript
declare const client: SMTPClient

client.sendAsync({
  text: '',
  from: '', // TS2345: Argument of type '{ text: string; from: string; to: string; cc: string; subject: string; }' is not assignable to parameter of type 'Message'.   Object literal may only specify known properties, and 'from' does not exist in type 'Message'.
  to: '', // same
  cc: '', // same
  subject: '' // same
})
```

And upon removing the four props that do not exist in type `Message`, the new error:

```
TS2345: Argument of type '{ text: string; }' is not assignable to parameter of type 'Message'.   Type '{ text: string; }' is missing the following properties from type 'Message': attachments, header, content, alternative, and 6 more.
```

The example should be something like

```typescript
const message = new Message({
  to, from, ...
})
client.sendAsync(message)
```

because, best as I can tell, the props are rightfully properties of the `headers` property of `Message`, not direct props of `Message`.

**Comment by zackschuster** [comment]
thanks for the catch! we actually support passing in the headers directly, so in this case the types are lying 😓 i've pushed a fix in 495d8fc8381c850c8ff400dea624c8e1510f1525 and will cut a release for it soon. cc @eleith

**Comment by eleith** [comment]
released under 3.8.0 on npm
